server/util/twilio.js

```diff
@@ -5,6 +5,10 @@ import Twilio from 'twilio';
 const twilioClient = new Twilio();
 const CALLER_ID = process.env.TWILIO_CALLER_ID;
 
-// Added the below console.logs to pass linting - please delete!
-console.log(twilioClient);
-console.log(CALLER_ID);
+export function sayHelloUser(userFirstName, campaignName) {
+  const VoiceResponse = Twilio.twiml.VoiceResponse;
+  const helloUser = new VoiceResponse();
+  helloUser.say(`Welcome, ${userFirstName}. You will be calling on behalf of ${campaignName}. Please follow the instructions on your computer screen to start calling.`);
+  helloUser.play({ loop: 0 }, 'https://api.twilio.com/cowbell.mp3');
```

Comment:
Oh god, let's not use cowbell as our hold music. Have you heard it? 😂 
How about something like http://com.twilio.music.classical.s3.amazonaws.com/ith_chopin-15-2.mp3.